<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OMEGAClass</class>
 <widget class="QMainWindow" name="OMEGAClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OMEGA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="omega.qrc">:/OMEGA/Images/OmegaMain.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_A">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Database:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Messaging:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SQLSTATUS">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : red; </string>
    </property>
    <property name="text">
     <string>Not Connected</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelQPIDSTATUS">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : red; </string>
    </property>
    <property name="text">
     <string>Not Connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_A_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Event Messaging:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_QpidEventStatus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : red; </string>
    </property>
    <property name="text">
     <string>Not Connected</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
             background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                               stop:0 lightgray, stop:1 darkgray);
         }
</string>
   </property>
   <widget class="QMenu" name="menuLauch_Component">
    <property name="title">
     <string>Lauch Component</string>
    </property>
    <addaction name="actionOption_Quoter"/>
   </widget>
   <widget class="QMenu" name="menuExit_OMEGA">
    <property name="title">
     <string>Exit OMEGA</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuManage_Component">
    <property name="title">
     <string>Manage Component</string>
    </property>
    <widget class="QMenu" name="menuOmega">
     <property name="title">
      <string>Omega</string>
     </property>
     <addaction name="actionStop_All_Quoters"/>
     <addaction name="separator"/>
     <addaction name="actionPause_All_Quoters"/>
    </widget>
    <addaction name="menuOmega"/>
   </widget>
   <addaction name="menuLauch_Component"/>
   <addaction name="menuManage_Component"/>
   <addaction name="menuAbout"/>
   <addaction name="menuExit_OMEGA"/>
  </widget>
  <action name="actionOption_Quoter">
   <property name="icon">
    <iconset resource="omega.qrc">
     <normaloff>:/OMEGA/Images/Optionquoter.png</normaloff>:/OMEGA/Images/Optionquoter.png</iconset>
   </property>
   <property name="text">
    <string>Option Quoter</string>
   </property>
  </action>
  <action name="actionStop_All_Quoters">
   <property name="icon">
    <iconset resource="omega.qrc">
     <normaloff>:/OMEGA/Images/stop.png</normaloff>:/OMEGA/Images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop All Quoters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPause_All_Quoters">
   <property name="icon">
    <iconset resource="omega.qrc">
     <normaloff>:/OMEGA/Images/pause.png</normaloff>:/OMEGA/Images/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause All Quoters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="omega.qrc"/>
  <include location="../WOPR/Wopr.qrc"/>
  <include location="../ExecEngine/ExecEngine.qrc"/>
  <include location="../Eclipse/Eclipse.qrc"/>
 </resources>
 <connections/>
</ui>
